src/compas/geometry/bbox/bbox_numpy.py

```diff
@@ -27,7 +27,7 @@
 
 
 def oriented_bounding_box_numpy(points):
-    """Compute the oriented minimum bounding box of a set of points in 3D space.
+    r"""Compute the oriented minimum bounding box of a set of points in 3D space.
```

Comment:
I hope you don’t mind me jumping in here, but I’m just curious why you make it a raw docstring? Is it an autodoc thing?